recipes/lcms2/meta.yaml

```diff
@@ -0,0 +1,53 @@
+package:
+  name: lcms2
+  version: 2.7
+
+source:
+  fn: lcms2-2.7.tar.gz
```

Comment:
To make updating the recipe simpler we usually use a jinja template for the version and then replace all instances of `2.7` with the template.